File: PHP/tabla_verdad.php
<?php
function tabla_verdad() {
    echo "Tabla de la verdad para la expresión (A AND B) OR (NOT A)\n";
    echo "A\tB\tResultado\n";
    $valores = [false, true];
    foreach ($valores as $A) {
        foreach ($valores as $B) {
            $resultado = ($A && $B) || (!$A);
            // Convertir booleanos a cadenas para mostrar
            $A_str = $A ? 'True' : 'False';
            $B_str = $B ? 'True' : 'False';
            $resultado_str = $resultado ? 'True' : 'False';
            echo "$A_str\t$B_str\t$resultado_str\n";
        }
    }
}

tabla_verdad();
?>

File: Java/tablaVerdad.java
public class tablaVerdad {
    public static void main(String[] args) {
        System.out.println("Tabla de la verdad para la expresión (A AND B) OR (NOT A)");
        System.out.println("A\tB\tResultado");
        
        boolean[] valores = { false, true };
        
        for (boolean A : valores) {
            for (boolean B : valores) {
                boolean resultado = (A && B) || (!A);
                System.out.println(A + "\t" + B + "\t" + resultado);
            }
        }
    }
}
